test/propagation.test.ts

```diff
@@ -24,7 +24,11 @@ import {
 } from '@opentelemetry/api';
 import { startTracing, stopTracing } from '../src/tracing';
 import { CompositePropagator, RandomIdGenerator } from '@opentelemetry/core';
-import { InMemorySpanExporter, SpanProcessor } from '@opentelemetry/tracing';
+import {
+  InMemorySpanExporter,
+  ReadableSpan,
```

Comment:
That's here from some experimentation, I guess, and the fact that nothing notifies us tells me we don't lint in CI.

Reply:
We do, I think this is `prettier` in action.

Reply:
prettier def doesn't add imports.